front/src/app/page.tsx

```diff
@@ -42,34 +41,15 @@ export default function Home() {
   });
 
   return (
-    <div className="flex flex-row justify-center">
-      <div className="pt-10 pb-20 px-10">
-        <div className="flex flex-row min-w-fit justify-center flex-nowrap text-nowrap">
-          <div className="basis-1/4 flex flex-col justify-center items-center">
```

Comment:
I think there's some magic in there which broke the page center alignment when being removed.